app/src/main/java/io/github/droidkaigi/confsched2017/receiver/NotificationReceiver.java

```diff
@@ -39,7 +37,6 @@ public void onReceive(Context context, Intent intent) {
             Timber.tag(TAG).v("Notification is disabled.");
             return;
         }
-        int sessionId = intent.getIntExtra(KEY_SESSION_ID, 0);
```

Comment:
🙋 
In my opinion, this should be here.
Because this notification will be improved to open each SessionDetailPage when the user tap the notification.
When we will implement to open SessionDetail, this `sessionId` is necessary.